packages/components/src/Provider/MarigoldProvider.tsx

```diff
@@ -19,11 +21,13 @@ export const MarigoldProvider: React.FC<ThemeProviderProps> = ({
   children,
 }) => {
   return (
-    <ThemeProvider theme={theme}>
-      <OverlayProvider>
-        {/* <GlobalStyles /> */}
-        {children}
-      </OverlayProvider>
-    </ThemeProvider>
+    <SSRProvider>
```

Comment:
This shouldn't be added here all the time, because we don't know if people use SSR by default!